package.json

```diff
@@ -61,7 +61,7 @@
     "test": "jest --verbose --silent -i",
     "test-emcc": "USE_EMCC=1 jest --verbose --silent",
     "ts": "yarn node --import tsx",
-    "build-plonky-2": "tsc -p beacon-light-client/plonky2/input_fetchers/tsconfig.json"
+    "build-plonky-2": "yarn tsc -b beacon-light-client/plonky2/input_fetchers/tsconfig.json"
```

Comment:
Seems like commit message is wrong... or is -b the same as --build? What was -p then?

Reply:
`-b` is short for `--build`. Used `build` in message to be clearer. 

> What was -p then

According to what I have read 
- The --project option ( -p ) compile a single project.
- The --build option can be seen as a build orchestrator that find referenced projects.

ref: https://stackoverflow.com/questions/67796765/tsc-build-vs-tsc-project